releasenotes/notes/improve-decomposition-controlled-one-qubit-unitaries-3ae333a106274b79.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features_circuits:
+  - |
+    Reduce the number of two-qubit gate counts when decomposing some multi-controlled
+    single-qubit unitary gates.
+    For example, for multi-controlled :class:`.YGate` on 10 qubits, we reduce the :class:`.CXGate`
+    count by 56%, for multi-controlled :class:`.HGate` on 10 qubits, we reduce the :class:`.CXGate`
+    count by 44%, for multi-controlled :class:`.SXGate` and :class:`.SXdgGate` on 10 qubits,
+    we reduce the :class:`.CXGate` count by 80%, and for multi-controlled :class:`.UGate` on 10 qubits,
+    we reduce the :class:`.CXGate` count by 31%.
```

Comment:
Would it make sense to show this data as a bullet list?

Reply:
done in [4823321](https://github.com/Qiskit/qiskit/pull/13801/commits/48233216b4af1649159b4dbf51d549e99be54e8f)